README.md

```diff
@@ -21,11 +21,10 @@ Due to complexity of F#'s compiler library, some features might not be available
 There are currently four targets for decompilation/disassembly:
 
 1. C#
-2. Visual Basic
-3. IL
-4. JIT Asm (Native Asm Code)
+2. IL
+3. JIT Asm (Native Asm Code)
 
-Note that C#=>VB or VB=>C# disassembly shouldn't be used to convert between languages, as the produced code is intentionally overly verbose.
+Note that C#=>VB disassembly shouldn't be used to convert between languages, as the produced code is intentionally overly verbose.

```

Comment:
Shouldn't you keep the other direction instead? I.e. since SharpLab can compile VB code and decompile it as C#, it could be in theory used to convert VB=>C#, but not C#=>VB.

Reply:
yes, sorry. I even proofread it and thought it was correct, haha. That's late night editing for you.